pkgs/development/python-modules/sphinx-autobuild/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, livereload
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-autobuild";
+  version = "2020.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vbaf4vrxahylyp8zrlw5dx1d2faajp926c3pl5i1wlkp1yll62b";
+  };
+
+  # No tests included.
+  doCheck = false;
```

Comment:
I would like to suggest to use the GitHub releases. The GitHub releases contains tests.

Reply:
Given how sparse the test suite is (one file basically https://github.com/executablebooks/sphinx-autobuild/blob/master/tests/test_ignore.py ). I don't see much of a benefit of building off of a git checkout over the sdist from PyPI.